packages/react/src/patterns/sections/CardSection/CardSectionImages.js

```diff
@@ -3,24 +3,27 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 /**
- * ImageCards pattern it is Cards with images
+ * CardSectionImages pattern it is Cards with images
  *
  * @param {object} props props object
  * @param {Array} props.cards cards array with title, and cards properties
  * @returns {object} JSX Object
  */
-const ImageCards = props => {
-  return <CardSection {...props} />;
+const CardSectionImages = props => {
+  const cardsWithImages = props.cards.filter(
+    card => card.image && card.eyebrow
+  );
+  return <CardSection {...(cardsWithImages && { ...props })} />;
 };
 
-ImageCards.propTypes = {
+CardSectionImages.propTypes = {
   theme: PropTypes.string,
-  title: PropTypes.string,
+  heading: PropTypes.string.isRequired,
   cards: PropTypes.arrayOf(
     PropTypes.shape({
-      image: PropTypes.object,
-      title: PropTypes.string,
-      copy: PropTypes.string,
+      image: PropTypes.object.isRequired,
+      eyebrow: PropTypes.string.isRequired,
+      heading: PropTypes.string.isRequired,
```

Comment:
if we add PropType.instanceOf(Card) then the all above prop validation is not required I believe. correct me here if I am wrong.

Reply:
I would agree with you except for the fact that `Card` is all optional by default. This might be one of the few places I'd encourage us to spell it out because we for these patterns some of the are now required.